.gitignore

```
@@ -16,6 +16,7 @@ server/dist
 junit-*
 .pytest_cache
 .idea
+*.cfcache/
```

It's always just `.cfcache`
```suggestion
.cfcache/
```